org.eclipse.mylyn.releng/multipass/mylyn_setup.yml

```diff
@@ -7,90 +7,28 @@
     - name: include global mode
       ansible.builtin.include_tasks:
         file: "global_setup.yml"
-
-#
-- hosts: localhost
-  gather_facts: false
-  roles:
-    - role: multipass_create_vm_setup
       tags:
-        - base_software
-# 
-#
-# Admin Instanz für Zertifikate und root ca 
-#
-- hosts: "{{ hostvars['localhost'].rootca_server_name }}"
-  gather_facts: false
-  vars:
-    rootca_name: "{{ hostvars['localhost'].rootca_server_name }}"
-  roles:
-    - role: crtrootca
-      vars:
-        common_domain_name: "{{ hostvars['localhost'].local_domain }}"
-      tags:
-        - createValidCertificates
-        - base_software
-    - role: crtcertificate
-      vars:
-        subject_alt: 
-          - "DNS:*.{{ hostvars['localhost'].local_domain }}"
-        common_domain_name: "{{ hostvars['localhost'].local_domain }}"
-      tags:
-        - createValidCertificates
-        - base_software
+        - global
+    - name: debug distribution
+      debug:
+        msg: "{{ localhostPlatform }}"
+
 #
-# Alle normalen Knoten mit Zertifikaten versorgen
+# Alle normalen Knoten Docker starten
```

Comment:
Deutsch ist gut fur Mich, aber ich denke das das nicht fur die Ganzen Welt gultet.

Reply:
I will fix this with the next commit.